pkg/watcher/skr_webhook_resources.go

```diff
@@ -17,17 +17,18 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 
+	"github.com/kyma-project/lifecycle-manager/api/shared"
 	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
 	"github.com/kyma-project/lifecycle-manager/pkg/log"
 )
 
 const (
-	podRestartLabelKey      = "operator.kyma-project.io/pod-restart-trigger"
+	podRestartLabelKey      = shared.OperatorPrefix + shared.Separator + "pod-restart-trigger"
```

Comment:
You can remove podRestartLabelKey fully, and also from `resources.yaml`. Got introduced a year back by Ali, has no usage currently and thus we could drop it.

Reply:
can you create an issue for this remove, it's not only remove this label, but have the actual [usage](https://github.com/kyma-project/lifecycle-manager/blob/c710461f7aa929c06f0d892f8dc251d9bda84625/pkg/watcher/skr_webhook_resources.go#L156), also that `cfg.secretResVer` should be cleanup, probably have more, but since you know more context, feel free add more detail in that issue.

Reply:
Yup, will create an issue for that